gateway-server/src/main/java/org/apache/knox/gateway/config/impl/GatewayConfigImpl.java

```diff
@@ -1094,4 +1100,14 @@ public boolean isTopologyValidationEnabled() {
 
     return set;
   }
+
+  @Override
+  public String getRefreshableServiceParametersFolder() {
```

Comment:
"Directory" is used, rather than "Folder", throughout the Knox code; I think we should maintain that consistency.